packages/malloy/src/runtime_types.ts

```diff
@@ -193,6 +196,10 @@ export interface StreamingConnection extends Connection {
   ): AsyncIterableIterator<QueryDataRow>;
 }
 
+export interface BrowsableSourceConnection extends Connection {
```

Comment:
This feels to me like a bit of a strange thing to include in our official API.

Is this something that other users of the Malloy library (other than VSCode) care about using?

If not, I would say this should be implemented in the VSCode extension. To do that, just move this interface into the VSCode extension and then instead of having the `Conection.browsableSource` method, have a function `isBrowsableSourceConnection(connection: Connection): connection is BrowsableSourceConnection`.

Reply:
removed